crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -76,23 +74,16 @@ impl Server {
     pub async fn run(self) -> Result<(), Error> {
         get_or_init_server_start_time().await;
 
-        // A handle that spawns a background task to periodically update the `CheckpointViewedAt`,
-        // which is the u64 high watermark of checkpoints that the service is guaranteed to produce
-        // a consistent result for.
+        // Initialize watch channel for epoch boundary.
+        let (tx, rx) = watch::channel(0);
```

Comment:
Create this when you create the `ServiceWatermarkTask` and offer an API to return a clone of the epoch signal receiver side from there, instead.